public/docs/ts/latest/guide/index.jade

```diff
@@ -60,55 +58,50 @@ table(width="100%")
         Authoritative details about each member of the Angular libraries.
 :marked
   # Learning path
-  We don't have to read the guide straight through.  Most chapters stand on their own.
+  You don't have to read the guide straight through.  Most chapters stand on their own.
 
-  We recommend a learning path for those new to Angular.
-  Most of that path runs through the *Basics* section:
+  For those new to Angular, the recommended learning path runs through the *Basics* section:
 
-  1. Read the [Architecture](architecture.html) overview to get the big picture.
+  1. For the big picture, read the [Architecture](architecture.html) overview.
 
-  1. Try the [QuickStart](../quickstart.html). The QuickStart is the "Hello, World" of Angular 2.
-  It shows us how to set up the libraries and tools we'll need to write *any* Angular app.
+  1. Try [QuickStart](../quickstart.html). QuickStart is the "Hello, World" of Angular 2.
+  It shows you how to set up the libraries and tools you'll need to write *any* Angular app.
 
-  1. Take the *Tour of Heroes* [Tutorial](../tutorial), which picks up from where the QuickStart leaves off
-  and builds a simple data-driven app.
-  Simple, yes, but with the essential characteristics we'd expect of a professional application:
+  1. Take the *Tour of Heroes* [tutorial](../tutorial), which picks up where QuickStart leaves off,
+  and builds a simple data-driven app. The app demonstrates the essential characteristics of a professional application:
   a sensible project structure, data binding, master/detail, services, dependency injection, navigation, and remote data access.
 
-    Return to the *Basics* section and continue in the suggested order:
+  1. [Displaying Data](displaying-data.html) explains how to display information on the screen.
 
-  1. [Displaying Data](displaying-data.html) explains how to get information on to the screen.
+  1. [User Input](user-input.html) covers Angular responds to user behavior.
 
-  1. [User Input](user-input.html) covers the basics of responding to user behavior.
+  1. [Forms](forms.html) handles user data entry and validation within the UI.
 
-  1. [Forms](forms.html) handle user data entry and validation within the UI.
-
-  1. [Dependency Injection](dependency-injection.html) is the way we build large, maintainable applications
+  1. [Dependency Injection](dependency-injection.html) is the way to build large, maintainable applications
   from small, single-purpose parts.
 
   1. [Template Syntax](template-syntax.html) is a comprehensive study of Angular template HTML.
 
-  With this foundation, we can read and understand any chapter in the guide.
+  These steps are a foundation, for understanding any chapter in the guide.
```

Comment:
"steps" seems wrong. Maybe:

Once you've read the above sections, you can skip around the rest of the guide.